### ISSUE TITLE
Unified CPU-GPU SoC listing

### ISSUE THREAD

**Issue by hyperjeff** [dylanaraps/neofetch#2318]
## Description

[Suggestion] Instead of showing two entries, CPU and GPU, that have the same information about, for example, the Apple M* SoC, consolidate this into a single "CPU-GPU" line. There will surely be (are?) other such vendor SoCs in the future as well. And then if there are also discrete GPUs, they could still be listed on a separate GPU line.

**Comment by 4r7if3x** [comment]
I agree, we have to show the APU information instead of the CPU & GPU on such systems.
